contracts/Strategy.sol

```diff
@@ -673,13 +603,11 @@ contract Strategy is BaseStrategy {
 
     function _calculateAmountToRepay(uint256 amount)
         internal
-        view
```

Comment:
why?

Reply:
removed then forgot to add back

good catch